Code fails to compile when using Warnings As Errors on the target

### Environment

How do you use Sentry?
Sentry.io 

Which version of the SDK?
Source code from git with libs manually integrated. 

Which version of Unreal?

5.0.0.

### Steps to Reproduce

1. Enable 	bWarningsAsErrors = true; in the target
2. Compile 

### Expected Result
Compile completes. 

### Actual Result

Compile fails: 
warning C6011: Dereferencing NULL pointer 'ptr'. : Lines: 92, 93

### Any logs or screenshots

--- a/plugin-dev/Source/Sentry/Private/SentryLibrary.cpp
+++ b/plugin-dev/Source/Sentry/Private/SentryLibrary.cpp
@@ -89,7 +89,10 @@ USentryAttachment* USentryLibrary::CreateSentryAttachmentWithPath(const FString&
 
 void USentryLibrary::Crash()
 {
+	// Supressing warnings for when using warnings as errors on the target. 
+	#pragma warning(suppress: 6011)	
 	char *ptr = 0;
+	#pragma warning(suppress: 6011)
 	*ptr += 1;
 }
 

